src/styles/_catalog.scss

```diff
@@ -1,22 +1,30 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$widthCard: 200px;
 
+@mixin responsive($minWidth, $howRepeat) {
```

Comment:
It's not obvious to use mixins like this
I call @include responsive, ok i pass $minWidth, it's ok. 
But $howRepeat does not applies to responsive. It's about grid. 
So what if i would like to have another 5-10 different styles that have to be changed inside responsive mixin?
Please, rewrite without using mixin like this.